app/src/main/java/no/usn/mob3000/ui/screens/auth/CreateUserScreen.kt

```diff
@@ -32,12 +32,12 @@ import no.usn.mob3000.ui.theme.DefaultButton
  * @param onSignUpClick Function to handle signing up a new account.
  * @param onReturnToLoginClick Callback function to navigate to [LoginScreen].
  * @author Markus
- * @contributor frigvid
+ * @contributor frigvid, Anarox
  * @created 2024-09-24
  */
 @Composable
 fun CreateUserScreen(
```

Comment:
Also, since you're using a similar view model function for registration like login, it's important to point out that you're not using it properly. Then you register, the screen receives no return from the view model, and doesn't use it in any way. No loading screen, no error message, and no success message.

In fact, testing it out, the problem of using the `AuthenticationViewModel` becomes apparent. That is designed for `login` specifically, even though `logout` leeches off of it, since it has no special requirements. Any error messages you get, are applied to the `LoginScreen`, which you can see by returning to that screen.